stravalib/client.py

```diff
@@ -291,9 +309,25 @@ def update_athlete(self, city=None, state=None, country=None, sex=None, weight=N
         https://developers.strava.com/docs/reference/#api-Athletes-updateLoggedInAthlete
 
         :param city: City the athlete lives in
```

Comment:
@jsamoocha are you open to migrating to numpy style docstrings? it seems to be a standard now and i think it's much easier to read. i started to implement this [back in this issue](https://github.com/stravalib/stravalib/issues/222#issuecomment-924501383) and had a local branch but that branch is super dated now . if you are open to it, when the time comes i'm happy to help with a bigger pr. i plan to do that with black anyway. But i wanted to run it by you first. i can create a specific issue about this.

there is a nice tool that does the conversion for you that I was using. and we don't have to implement it in this PR it's just something i've been thinking about.

Reply:
Sure, especially if there's a tool out there that will save us from doing it all manually!

Reply:
there is! we can work on that in another PR. i found this great command line tool that automagically converts docstring format :) yay!